.gitignore

```diff
@@ -12,4 +12,5 @@
 *.out
 
 # Dependency directories (remove the comment below to include it)
-# vendor/
+# vendor/.DS_Store
```

Comment:
should `.DS_Store` be in a new line? It is not a sub folder of the vendor, right?